docs/adding_packages/dependencies.md

```diff
@@ -0,0 +1,20 @@
+# Dependencies
+
+This section outlines all the practices and guidelines for `requirements()` and `build_requirements()` methods. This includes everything for "vendored" dependencies to
+when and how the versions could be changes.
+
+<!-- toc -->
+## Contents
+
+  * [Rules](#rules)<!-- endToc -->
+
+## Rules
+
+* Version range is not allowed.
+* Specify explicit RREV (recipe revision) of dependencies is not allowed.
+* Vendoring in library source code should be removed (best effort) to avoid potential ODR violations. If upstream takes care to rename
+  symbols, it may be acceptable.
+* Only other ConanCenter recipes are allowed in `requires`/`requirements()` and `build_requires`/`build_requirements()`.
```

Comment:
nope, it's also allowed to reference the recipe itself in build_requirements

Reply:
Did you leave this comment on the wrong line? Your comment seems to been a touch off topic 🤔 This is junk left over from then bintray existed.

Reply:
nope, it's for exactly that line
it says `Only other recipe` can be referenced in `build_requires`. in fact, the **same** recipe might be perfectly referenced in `build_requires`:
https://github.com/conan-io/conan-center-index/blob/master/recipes/icu/all/conanfile.py#L85-L86
it's legit and legal

Reply:
```suggestion
* Only ConanCenter recipes are allowed in `requires`/`requirements()` and `build_requires`/`build_requirements()`.
```